public/botViews/popUpSignin.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>    
+</head>
+<!-- This simulates a login page to show what happens when the user clicks on a button with the signin action.
+When the user clicks on the Login button, it calls notifySuccess() to fake a successful login and trigger a new message to the bot -->
+<body>
+    <div>
+        User Name : <input type="text" />
+    </div>
+    <br />
+    <div>
+        Password : <input type="password" />
+    </div>
+
+    <button onClick="submit()">Login</button> 
+
+    <script src="https://statics.teams.microsoft.com/sdk/v1.0/js/MicrosoftTeams.min.js"></script>
+
+    <script>
+        microsoftTeams.initialize();
+        function submit()
+        {
+            microsoftTeams.authentication.notifySuccess('PopUpSignInAuthenticationSuccess');
```

Comment:
For this example, let's pass the user's input username and password through the notifySuccess call so that the verifyState handler in the bot can present those fields in the outgoing message.